SUI/SUI/ContentView.swift

```diff
@@ -4,15 +4,182 @@
+                                .foregroundColor(.red)
+                            Text(Constants.oneString)
+                                .foregroundColor(.white)
+                        }
+                        chewron()
+                    }
+                    HStack {
+                        Text(Constants.memoryText)
+                        Spacer()
+                        ZStack {
+                            Circle().frame(width: Constants.circleWidth)
+                                .foregroundColor(.red)
+                            Text(Constants.oneString)
+                                .foregroundColor(.white)
+                        }
+                        chewron()
+                    }
+                }
+                
+                Section {
+                    HStack {
+                        Image(Constants.aviaImageName).cornerRadius(Constants.imageCornerRadius)
+                        Text(Constants.aviaText)
+                        Toggle(isOn: $isOnAvia) {}
+                    }
+                    HStack {
+                        Image(Constants.wifiImageName).cornerRadius(Constants.imageCornerRadius)
+                        Picker(selection: $wifiIndex) {
+                            ForEach(0..<wifis.count) {
+                                Text(self.wifis[$0])
+                            }
+                        } label: {
+                            Text(Constants.wifiText)
+                        }
+                    }
+                    HStack {
+                        Image(Constants.bluetoothImageName).cornerRadius(Constants.imageCornerRadius)
+                        Picker(selection: $blueToothIndex) {
+                            ForEach(0..<bluTooths.count) {
+                                Text(self.bluTooths[$0])
+                            }
+                        } label: {
+                            Text(Constants.bluetoothTitle)
+                        }
+                    }
+                    HStack {
+                        Image(Constants.cellImageName).cornerRadius(Constants.imageCornerRadius)
+                        Text(Constants.cellTitle)
+                        Spacer()
+                        chewron()
+                    }
+                    HStack {
+                        Image(Constants.modemImageName).cornerRadius(Constants.imageCornerRadius)
+                        Text(Constants.modemTitle)
+                        Spacer()
+                        chewron()
+                    }
+                    HStack {
+                        Image(Constants.vpnImageName).cornerRadius(Constants.imageCornerRadius)
+                        Text(Constants.vpnTitle)
+                        Toggle(isOn: $isOnVPN) {}
+                    }
+                }
+            }.navigationTitle(Constants.settingsTitle)
         }
-        .padding()
+    }
+    
+    fileprivate func chewron() -> some View {
```

Comment:
```suggestion
    private var chewronView: some View {
```
если нет параметров делай не метод а вычисляемое свойство